besu/src/main/java/org/hyperledger/besu/cli/options/JsonRpcHttpOptions.java

```diff
@@ -394,10 +411,11 @@ private void validateTls(final CommandLine commandLine) {
 
     if (isRpcHttpTlsClientAuthEnabled
         && !isRpcHttpTlsCAClientsEnabled
-        && rpcHttpTlsKnownClientsFile == null) {
+        && rpcHttpTlsKnownClientsFile == null
+        && rpcHttpTlsTruststoreFile == null) {
       throw new CommandLine.ParameterException(
           commandLine,
-          "Known-clients file must be specified or CA clients must be enabled when TLS client authentication is enabled for JSON-RPC HTTP endpoint");
+          "Known-clients file or Truststore file must be specified or CA clients must be enabled when TLS client authentication is enabled for JSON-RPC HTTP endpoint");
```

Comment:
what happens if both are specified? should it be one or the other?

Reply:
Yeah, it should be one or the other . Added option dependency checks to allow either 1) Known-clients file and/or CA clients(default truststore) 2) Custom truststore